### ISSUE TITLE
Agenda view in calendar

### ISSUE THREAD

**Issue by MadsSkaaning** [w11k/calendar-for-trello#128]
It would be nice with a agenda view in calendar. (in fact, totally as the Google Calendar "Agenda" view)
(https://calendar.google.com/calendar/render?pli=1#main_7%7Clist for logged in users)

**Comment by vcakarov** [comment]
Yes and also to display it for a selected member (related to issue #117).

**Comment by jblankenhorn** [comment]
+1

**Comment by ededagic** [comment]
+1

**Comment by alekwo** [comment]
+1
